genesyscloud/provider.go

```diff
@@ -282,7 +280,7 @@ func initClientConfig(data *schema.ResourceData, version string, config *platfor
 		},
 		ResponseLogHook: func(response *http.Response) {
 			if response.StatusCode < http.StatusOK || response.StatusCode >= http.StatusMultipleChoices {
-				log.Printf("Response %s", response.Status)
+				log.Printf("Response %s for request:%s %s", response.Status, response.Request.Method, response.Request.URL)
```

Comment:
Added the url in the response log so if we get a non-200 response we no longer get a little more info.